src/lib/Behat/Page/ContentTypeUpdatePage.php

```diff
@@ -123,4 +125,10 @@ public function verifyIsLoaded(): void
         parent::verifyIsLoaded();
         $this->getHTMLPage()->find($this->getLocator('contentTypeAddButton'))->assert()->isVisible();
     }
+
+    public function selectEditorLaunchMode(string $viewMode): void
+    {
+        $this->getHTMLPage()->findAll($this->getLocator('selectLaunchEditorMode'))
+            ->getByCriterion(new ElementAttributeCriterion('value', $viewMode))->click();
```

Comment:
```suggestion
        $this->getHTMLPage()
             ->findAll($this->getLocator('selectLaunchEditorMode'))
             ->getByCriterion(new ElementAttributeCriterion('value', $viewMode))
             ->click();
```

Nitpick, but for more it's more readable to split it this way (when the code is too long to fit in one line)